Ascend.java

```diff
@@ -2,6 +2,26 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		// get input number for a limit value and random 3 numbers between 0 and the input
+		int number=Integer.parseInt(args[0]);
+		number=number+1;
+		double num1=Math.random()*number;
+		double num2=Math.random()*number;
+		double num3=Math.random()*number;
+		//save the values in another variables
+		int num4=(int)num1;
+		int num5=(int)num2;
+		int num6=(int)num3;
+		//check which number is the min, max and for the middle number i need to subtract the min and the max 
+		//from the sum of the numbers.
+		int min= Math.min(num4,num5);

```

Comment:
**can use:**
int min = Math.min(Math.min(num1, num2), num3)
instead of:
int min= Math.min(num1,num2);
min = Math.min(min, num3)